Add license / copyright info to README.md (#5)
ccmmf/.github: profile/README.md

This section will be developed as the project gets under way. In the mean time, feel free to reach out directly to team members or through our [GitHub
Discussions](https://github.com/orgs/ccmmf/discussions).

## Licenses and Copyright

> A central focus of this project is to generate open source software and open data that will enable transparency while facilitating reuse, collaboration, and derivative works. To this end, we plan to release our works under the following licenses:

Reviewer: ```suggestion
A central focus of this project is to generate open source software and open data that will enable transparency while facilitating reuse, collaboration, and derivative works. To this end, we release our works under the following licenses:
```

> - Software: BSD 3-Clause or compatible license

Reviewer: Do we need the "or compatible" here?

Author: This is a good point - the motivation was that some of our code will be released as part of pecan, which hasn't been completely relicensed.

Reviewer: Since PEcAn is a separate project (and thus not "our works") with its own existing licensing, I don't think that needs a disclaimer when talking about CCMMF's org-level policies.

Author: ```suggestion
- Software: BSD 3-Clause 
```

- Writing: [CC-BY](https://creativecommons.org/licenses/by/4.0/)

> - Data Products: Public domain ([CC0](https://creativecommons.org/public-domain/cc0/)).

Reviewer: ```suggestion
- Data Products: Public domain ([CC0](https://creativecommons.org/public-domain/cc0/)).

If any work in a CCMMF repo is missing its LICENSE file or its terms of use are otherwise not clear, please tell us -- that is a bug and we will fix it.
```

Author: I don't think this is necessary, since any repository missing a LICENSE file will inherit https://github.com/ccmmf/.github/blob/main/LICENSE

Reviewer: TBH this is more to be reassuring to the reader than for us -- "we just showed you a list, but there will not be a quiz. It's not your job to figure out whether we thought a given file was 'writing' or 'data' for licensing purposes."

This section will be developed as the project gets under way. In the mean time, feel free to reach out directly to team members or through our [GitHub
Discussions](https://github.com/orgs/ccmmf/discussions).

## Licenses and Copyright

A central focus of this project is to generate open source software and open data that will enable transparency while facilitating reuse, collaboration, and derivative works. To this end, we release our works under the following licenses:
   

> - Software: BSD 3-Clause 

Author: ```suggestion
- Software: [BSD 3-Clause](https://opensource.org/license/bsd-3-clause) 
```

- Writing: [CC-BY](https://creativecommons.org/licenses/by/4.0/)
- Data Products: Public domain ([CC0](https://creativecommons.org/public-domain/cc0/)).

> If any work in a CCMMF repo is missing its LICENSE file or its terms of use are otherwise not clear, please tell us -- that is a bug and we will fix it.

Author: ```suggestion
If any CCMMF software repositories, data products, or documentation are missing a LICENSE file or if their terms of use are unclear, please let us know. For any questions about the terms of use, feel free to [start a discussion](https://github.com/orgs/ccmmf/discussions) or contact a team lead.
```

Author: Just revised to remove 'we consider this a bug'...
